doc/classes/XRHandTracker.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="XRHandTracker" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
How about classifying it as experimental just in case?
```suggestion
<class name="XRHandTracker" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd" experimental="">
```

Reply:
Perhaps. But I don't know if it makes sense to mark this as experimental if we're also deprecated the older APIs. I feel like we should do one or the other: Mark the old APIs as deprecated OR mark the new APIs as experimental, but not both.

Reply:
Hmm, I think it's overkill, probably the original OpenXR implementation should have been called experimental, but since this is all based on the experience we gathered with this, I think we have a pretty solid grasp on it.